web-wallet/package.json

```diff
@@ -4,17 +4,17 @@
 		"url": "https://dusk.network/",
 		"mail": "info@dusk.network"
 	},
-	"bugs": "https://github.com/dusk-network/web-wallet/issues",
+	"bugs": "https://github.com/dusk-network/rusk/issues",
 	"engines": {
 		"node": ">=18.0.0"
 	},
-	"homepage": "https://github.com/dusk-network/web-wallet",
+	"homepage": "https://github.com/dusk-network/rusk/tree/master/web-wallet",
 	"license": "MPL-2.0",
 	"name": "web-wallet",
 	"private": true,
```

Comment:
This brought my attention. I think it's fine, just wanted to confirm.

NodeJS documentation says:
`If you set "private": true in your package.json, then npm will refuse to publish it.` which, I guess, is fine, as we don't want to have this published as a module independently, right? 

But for the wallet.dusk.network – are we still going to deploy this Wallet, which is under Rusk (my understanding is yes, otherwise it will be a mess maintaining two different versions of the Wallet). I'm not sure how this deployment relates to the private property in the json file.

I guess it's good time to learn. 🙈 Thanks!

Reply:
The publishing refers to npm registries, it's not about deploying.

Reply:
Yeah, yeah. I am aware of that. Was just wondering if the property makes much sense in our case.

Reply:
To avoid accidentally publishing it on someone's npm account.